docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -106,37 +130,34 @@ This creates a tarball of the built collection in the current directory which ca
     └── ...
 
 .. note::
-   * Certain files and folders are excluded when building the collection artifact. See :ref:`ignoring_files_and_folders_collections`  to exclude other files you would not want to distribute.
-   * If you used the now-deprecated ``Mazer`` tool for any of your collections, delete any and all files it added to your :file:`releases/` directory before you build your collection with ``ansible-galaxy``.
+   * To reduce the size of collections, certain files and folders are excluded from the collection tarball by default. See :ref:`ignoring_files_and_folders_collections` if your collection directory contains other files you want to exclude.
+   * If you used the now-deprecated ``Mazer`` tool for any of your collections, delete all files that Mazer added to your :file:`releases/` directory before you build your collection with ``ansible-galaxy``.
```

Comment:
Do we still need the Mazer refs?  It's been EOL since Ansible 2.9 was released.

Reply:
Fixed in 2b84951fc61422551f159ccf5df686beff5fb23e.